nilearn/plotting/img_plotting.py

```diff
@@ -649,6 +650,7 @@ def plot_prob_atlas(maps_img, anat_img=MNI152TEMPLATE, view_type='auto',
                     output_file=None, display_mode='ortho',
                     figure=None, axes=None, title=None, annotate=True,
                     draw_cross=True, black_bg='auto', dim=False,
+                    colorbar=True,
```

Comment:
I think that colorbar should be False by default (and in the example just add one being "True").